components/Footer/Footer.tsx

```diff
@@ -19,13 +25,15 @@ export const Footer = () => {
   const intl = useIntl()
   return (
     <FooterWrapper>
-      <Container>
-        <StyledFooter>
-          <Typography variant="body_short">
-            {intl.formatMessage({ id: "footer.copyright" })}
-          </Typography>
-        </StyledFooter>
-      </Container>
+      <TopBorderBox>
+        <Container>
+          <StyledFooter>
+            <Typography variant="body_short">
+              {intl.formatMessage({ id: "footer.copyright" })}
+            </Typography>
+          </StyledFooter>
+        </Container>
+      </TopBorderBox>
```

Comment:
lot of `div`-ing going on right now. imo be better if we moved `<Footer />` to the `<Page />`, set a default `margin-top` to the footer, set some global css to make body `display: flex|grid` and have `main|article` take up the space it needs to fill the page:

```css
body {
  display: grid;
  grid-template-rows: auto 1fr auto;
}
/* or */
body {
  display: flex;
  flex-direction: column;
}

main, article {
  flex: 1;
}
```

but we can look at that at a later time if you feel like it's out of scope for this pr.

Reply:
Det er helt riktig observert! 😅 

Men jeg tenker at vi kan hanke inn igjen denne som en del av [denne tasken](https://dev.azure.com/EquinorASA/Data%20to%20the%20Many/_workitems/edit/78543/). 

Jeg liker den teknikken jeg har brukt fordi den er ganske fleksibel med tanke på hva andre ting man har på siden. Det som gjør det litt knotete er også den Container-klassen som jeg brukte her, skulle kanskje ikke gjort det fordi da kunne vi bare lagt på spacing i Container heller. Ideelt sett ville jeg hatt spacingen (margin + padding) på sidene som CSS-variabler, helst fluid, sånn at vi kunne brukt variabelen i Container og i footeren direkte

Reply:
På Envis bruker vi forresten with layout-pattern til Next.js, så vi kunne selvsagt tatt inn det istedenfor og vente på nye Next.js-features. Men jeg tenker at det ikke er nødvendig, ettersom hvordan man gjør det, endrer seg mye 😄

Reply:
vi bruker `Page` for layout akkurat nå. trenger ikke å dra inn noe annet